plugins/talk-plugin-featured-comments/client/index.js

```diff
@@ -76,24 +90,27 @@ export default {
     RemoveTag: ({variables}) => ({
       updateQueries: {
         CoralEmbedStream_Embed: (previous) => {
-
+          let updated = previous;
+           
           if (variables.name !== 'FEATURED') {
             return;
           }
 
-          const updated = update(previous, {
-            asset: {
-              featuredComments: {
-                nodes: {
-                  $apply: (nodes) =>
-                    nodes.filter((n) => n.id !== variables.id)
+          if (previous.asset.comments) {
```

Comment:
Same here. It was triggering an error in the permalink view. (Where we don't have `comments`)

Reply:
Removing this should not produce an error, at least I can't reproduce it.

Reply:
Ah ok now I know what you mean, checking..

Reply:
Do you have the permalink plugin enabled? Because it makes sense that it tries to access `comments` and `comments`it's not there (only `comment`)

@IAmSamHankins reported this and she was able to reproduce this. Same happened to me.

Reply:
Yeah this one makes sense 👍